### ISSUE TITLE
Your Readme lacks clarity

### ISSUE THREAD

**Issue by funkytaco** [diablodale/jit.openni#23]
Hi, I wandered here from a blog post. I'm running OSX and trying to use Kinect with some sort of particle system for an art installation at my wedding.

I installed all the software when I realized two things:
- There were no instructions for the install of jit.openni
- I didn't have the "Max" software.

Now, "Max" is a pretty hard term to google , especially if the company's website is nonsensical. I take it I'd need to buy this software?  http://cycling74.com/products/

In other words, do I need that Max software? If so, please mention it as a prerequisite. I take it once it's installed, what to do with jit.openni will be clear? 

Thank you!

**Comment by diablodale** [comment]
Thank you for the feedback. I do have some follow concerns:
1. The README does have reference to full install instructions. Please see the top of the README where is written, "Full install and usage documentation at...". Also on the webpage http://hidale.com/jit-openni/ is a second labeled "Documentation" and under it similar text written, "Full wiki documentation and installation instructions..." I also repeat the OSx installation instructions themselves in the README and the linked locations. In the REAME, it is, "Mac OSx install notes:". In the linked location (the wiki) it is "Download and Install" and "Additional Mac OSx install notes". If these locations are not good enough, I welcome specific feedback on where you think I should put it or a better way to write it.
2. It is true that I only wrote "Max 6.0.7 or higher". Thank you for the feedback, I have updated the text to be "Cycling 74 Max 6.0.7 or higher"
3. Yes, this external (a term for an extension/enhancement for Cycling 74's Max) does require the Cycling 74 Max software. From what blog post did you wander here? Perhaps if I see that, I can better imagine how to write the documentation for people that have little to no knowledge on the topic.

Thank you again for the feedback. I look forward to your clarifications.

**Comment by diablodale** [comment]
closing due to no response
